components/textbook-beta/TextbookBetaHeader.vue

```diff
@@ -102,16 +105,18 @@ export default class TextbookBetaHeader extends (Vue as VueConstructor<VueCompon
     @include contained();
 
     max-width: $max-size;
-    background-image: url("/images/textbook-beta/qiskit-logo-header.png");
+    //background-image: url("/images/textbook-beta/qiskit-logo-header.png");
```

Comment:
this should be safe to remove @korgan00

Reply:
I removed other background-related properties.